### ISSUE TITLE
Add a lightbox component 

### ISSUE THREAD

**Issue by Tanuj22** [layer5io/layer5#1627]
**Desired Behavior**
A lightbox component is needed. 

---
[Optional] **Screenshots / Mockups**

[Optional] **Alternatives**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

**Comment by UnkleTayo** [comment]
@Tanuj22 is this issue open?

**Comment by Tanuj22** [comment]
I was going to work on it but you can go ahead. Use this https://www.npmjs.com/package/react-image-lightbox

**Comment by UnkleTayo** [comment]
Alright, is there a design for this? @Tanuj22

**Comment by Tanuj22** [comment]
> Alright, is there a design for this? @Tanuj22

No. We just need a lightbox so that images can be seen  in enlarged view

**Comment by UnkleTayo** [comment]
> > Alright, is there a design for this? @Tanuj22
> 
> No. We just need a lightbox so that images can be seen in enlarged view

So I'm only creating a component for that right?

**Comment by Tanuj22** [comment]
I think this library can directly be used on an image. If so, you can show can example on the some of the images in the blog posts

**Comment by UnkleTayo** [comment]
Okay, I'll try and see how it works

**Comment by Tanuj22** [comment]
@UnkleTayo any update here?
